src/tyr/isochrone_serializer.cc

```diff
@@ -2,12 +2,17 @@
 #include "baldr/json.h"
 #include "midgard/point2.h"
 #include "midgard/pointll.h"
+#include "thor/worker.h"
 #include "tyr/serializers.h"
 
 #include <cmath>
 #include <sstream>
 #include <utility>
 
+#ifdef ENABLE_GDAL
+#include <gdal/gdal_priv.h>
```

Comment:
i thought the same thing but the examples show it for the c++ types: https://gdal.org/tutorials/raster_api_tut.html

it might be a similar situation to geos though wehre the c api is stable and the c++ one less so. i havent checked on gdal use in c++ since long long ago when i first used it in valhalla for elevation datasets

Reply:
uff I don't hope it's the same as geos haha that was a huge PITA. but I doubt it, geos was flagging that in capital letters in their readme, gdal doesn't seem to do that.